cmd/thanos/receive.go

```diff
@@ -439,6 +439,10 @@ func runReceive(
 	{
 		var s *grpcserver.Server
 		startGRPC := make(chan struct{})
+
+		// set the request logging to no logging
+		requestLoggingDecision := "NoLogCall"
```

Comment:
Does it has to be string? Ideally we can use constant (:

Reply:
I am using `string` because it is compatible with the datatype of the flags option that we receive.